openmetadata-ui/src/main/resources/ui/src/pages/tour-page/TourPage.component.tsx

```diff
@@ -101,17 +99,13 @@ const TourPage = () => {
   useEffect(() => {
     handleIsTourOpen(true);
     AppState.currentTourPage = CurrentTourPageType.MY_DATA_PAGE;
-    AppState.activeTabforTourDatasetPage = 1;
+    AppState.activeTabforTourDatasetPage = EntityTabs.SCHEMA;
```

Comment:
I would recommend creating an action for it instead of directly updating the state.

Reply:
Will cover this in my next PR